packages/unlock-js/src/PublicLock/utils/getPurchaseKeysArguments.js

```diff
@@ -0,0 +1,54 @@
+import formatKeyPrice from './formatKeyPrice'
```

Comment:
Since this is versioned (there is a v10 version) can we please just use a version for this rather than in the `utils` folder? This way we remain consistent and we can cleanup more easily.

Reply:
It is not versioned.